src/main/java/me/zeroeightsix/kami/KamiMod.java

```diff
@@ -34,23 +34,19 @@
 import java.net.URL;
 import java.nio.charset.Charset;
 
-/**
- * Created by 086 on 7/11/2017.
- * Updated by l1ving on 25/03/19
- * Updated by Dewy on 09/04/2020
- */
 @Mod(
-        modid = KamiMod.MODID,
-        name = KamiMod.MODNAME,
-        version = KamiMod.VER_FULL_BETA
+        modid = KamiMod.ID,
+        name = KamiMod.NAME,
+        version = KamiMod.VERSION
 )
 public class KamiMod {
 
-    public static final String MODNAME = "KAMI Blue";
-    public static final String MODID = "kamiblue";
-    public static final String VER_FULL_BETA = "v1.1.7-beta"; // this is changed to v1.x.x-commit for debugging by automatic builds
-    public static final String VER_SMALL = "v1.1.7-beta"; // shown to the user, unchanged
-    public static final String VER_STABLE = "v1.1.6"; // used for update checking
+    public static final String NAME = "KAMI Blue";
+    public static final String ID = "kamiblue";
+    public static final String VERSION = "1.11.xx-dev"; // Used for debugging. R.MM.DD-hash format.
+    public static final String VERSION_SIMPLE = "1.11.xx-dev"; // Shown to the user. R.MM.DD[-beta] format.
+    public static final String VERSION_MAJOR = "1.11.01"; // Used for update checking. RR.MM.01 format.
```

Comment:
Why not just get this from `VERSION` with some string replacing. Less work for updating

Reply:
No, it's updated automatically.